### DISCUSSION TITLE
does anyone know how to build and test [Jul 19]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2844] [Discord Threads]
> 🙋‍♂️ **Sabishii**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Sabishii**  
#### nvm i figured it out by following this https://docs-dev.nocodb.com/engineering/development-setup/  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/998565831967506462)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
